packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -34,13 +33,11 @@
   "force_lightning_event_create_text": "SFDX: Create Lightning Event",
   "force_lightning_interface_create_text": "SFDX: Create Lightning Interface",
   "force_source_status_local_text": "SFDX: View Local Changes",
-  "force_source_status_remote_text":
-    "SFDX: View Changes in Default Scratch Org",
+  "force_source_status_remote_text": "SFDX: View Changes in Default Org",
```

Comment:
I would suggest leaving "Scratch" in the messages on lines 10 through 36 (`force_source_pull_default_scratch_org_text` through `force_source_status_remote_text`). Scratch orgs are the only orgs with change tracking, so they're the only orgs these commands will work with.